nservicebus/hosting/nservicebus-host/logging-configuration.md

```diff
@@ -12,14 +12,14 @@ include: host-deprecated-warning
 
 ## Logging configuration
 
-This article explains how to customize default configuration. Refer to the [Profiles - Logging](/nservicebus/hosting/nservicebus-host/profiles.md#logging) section for more details about logging configuration in built-in profiles.
+This article explains how to customize the logging configuration when using the NServiceBus Host. For more details about logging configuration in built-in profiles, refer to the [NServiceBus.Host Profiles - Logging](profiles.md#logging) section.
```

Comment:
> For more details about logging configuration in built-in profiles

That doesn't apply to the latest version, correct? As we don't have profiles anymore. So that should only appear in partials for earlier versions then. Which is the last version that has profiles?